Gem/Code/Source/Components/Multiplayer/GemComponent.cpp

```diff
@@ -32,15 +32,25 @@ namespace MultiplayerSample
 
             // Tick on every frame.
             m_clientAnimationEvent.Enqueue(AZ::Time::ZeroTimeMs, true);
+
+            Physics::RigidBodyRequestBus::Event(GetEntityId(), &Physics::RigidBodyRequestBus::Events::DisablePhysics);
+            // Physical bodies take time to enable after entity activation, so sign up for physics activation and disable it
+            Physics::RigidBodyNotificationBus::Handler::BusConnect(GetEntityId());
```

Comment:
You can remove `Physics::RigidBodyRequestBus::Event(GetEntityId(), &Physics::RigidBodyRequestBus::Events::DisablePhysics);`

Connecting to the `RigidBodyNotificationBus` bus will call you directly to `OnPhysicsEnabled` if it was already activated, if not then it'll call you later. So you can remove the first `DisablePhysics`

Reply:
Done